rundeckapp/grails-spa/packages/ui-trellis/src/app/components/activity/activityList.vue

```diff
@@ -301,9 +301,9 @@
                   <span v-else>
                     {{rpt.executionString}}
                   </span>
-                  <span v-if="query.jobIdFilter && rpt.jobId && query.jobIdFilter !==rpt.jobId && query.jobIdFilter !=='!null'" class="text-secondary">
+                  <span v-if="query.jobIdFilter && rpt.jobId && query.jobIdFilter !==rpt.jobId && query.jobIdFilter !=='!null' && rpt.jobUuid" class="text-secondary">
```

Comment:
This if statement should just check the rpt.jobUuid against the query.jobIdFilter instead of adding the uuid to the if.

Something like: `v-if="query.jobIdFilter && rpt.jobUuid && query.jobIdFilter !==rpt.jobUuid && query.jobIdFilter !=='!null'"`